dimod/sampleset.py

```diff
@@ -1178,16 +1218,17 @@ def lowest(self, rtol=1.e-5, atol=1.e-8):
                           self.vartype)
 
     def truncate(self, n, sorted_by='energy'):
-        """Create a new SampleSet with rows truncated after n.
+        """Create a new sample set with up to n rows.
```

Comment:
I am worried that it is not clear that it takes the first n rows.

Reply:
I feel that with both (1) the function name, "truncate" and (2) the following description, "Does not return  any rows above this limit in the original" that the error probability is sufficiently low. If you are highly worried, I can change